example/src/Views/Checkout/PaymentMethodsView.js

```diff
@@ -10,26 +10,32 @@ import {
   Image,
   Text,
   useColorScheme,
+  Alert,
 } from 'react-native';
 import Styles from '../../Utilities/Styles';
-import { ENVIRONMENT } from '../../Configuration';
+import {ENVIRONMENT} from '../../Configuration';
+import {payByID} from '../../Utilities/payByID';
+import {useAppContext} from '../../Utilities/AppContext';
 
-const PaymentMethods = () => {
-  const { start, paymentMethods: paymentMethodsResponse } = useAdyenCheckout();
+const PaymentMethods = ({isSession}) => {
+  const {configuration} = useAppContext();
+  const {start, paymentMethods: paymentMethodsResponse} = useAdyenCheckout();
   const regularPaymentMethods = paymentMethodsResponse?.paymentMethods ?? [];
   const storedPaymentMethods = paymentMethodsResponse?.storedPaymentMethods;
 
   const isNotReady = paymentMethodsResponse === undefined;
   const isDarkMode = useColorScheme() === 'dark';
 
-  const subtitle = (/** @type {import('@adyen/react-native').StoredPaymentMethod} */ pm) => {
+  const subtitle = (
```

Comment:
It could be simplified as:
```javascript
  const subtitle = (
    /** @type {import('@adyen/react-native').StoredPaymentMethod} */ pm
  ) =>
    pm.type === 'scheme'
      ? `**** **** **** ${pm.lastFour} (exp ${pm.expiryMonth}/${pm.expiryYear})`
      : `${pm.id}`;
```

Reply:
It could be useful when we have stored Cashapp and stored Blik support.